src/test/java/kitchenpos/application/ServiceTest.java

```diff
@@ -0,0 +1,18 @@
+package kitchenpos.application;
+
+import kitchenpos.acceptance.DatabaseCleanup;
+import org.junit.jupiter.api.BeforeEach;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+public class ServiceTest {
```

Comment:
```suggestion
public abstract class ServiceTest {
```

소소하지만.. abstract를 통해 의도를 나타낼 수 있겠네요 ㅎㅎ